app/public/js/system/settings.js

```diff
@@ -2,7 +2,19 @@
 
 const ua = require('universal-analytics');
 const fs = require('fs');
-const userConfig = JSON.parse(fs.readFileSync(`${__dirname}/userConfig.json`, 'utf-8'));
+const userHome = require('user-home');
+
+let _userConfigPath = `${__dirname}/app/public/js/system/userConfig.json`; // Windows specific for now
```

Comment:
Let's export this path config to a util function that returns the `_userConfigPath `